wiki/en/Server-Troubleshooting.md

```diff
@@ -19,25 +19,25 @@ permalink: "/wiki/Server-Troubleshooting"
 
 </details>
 
-## Servers - Registered 
+## Servers - Registered
 
 ### Why doesn't my Server show up in the list? Why isn't it registering?
 
-If you are registered OK (you can [see it here](https://explorer.jamulus.io/)) and you or your friends can't see your Server, you may need to wait, or start your Client with the `--showallservers` option and try connecting from there ([see command line options](Software-Manual#command-line-options)) on how to start your Client with a config option). In some network configurations, you may also need to [port forward](Running-a-Server#port-forwarding) your router.
+If you are registered OK (you can [see it here](https://explorer.jamulus.io/)) and you or your friends can't see your Server, you may need to wait, or start your Client with the `--showallservers` option and try connecting from there ([see command line options](Software-Manual#command-line-options)) on how to start your Client with a config option). In some network configurations, you may also need to [port forward](Unregistered-Servers#port-forwarding) your router.
```

Comment:
`--showallservers` does not appear on [the current Software-Manual page](https://jamulus.io/wiki/Software-Manual#command-line-options).

Reply:
We thought it better to leave it undocumented to avoid confusion (long story, but it's not what it seems). It's just for debugging.